index.html

```diff
@@ -3635,6 +3643,42 @@ <h4>Protocol Binding based on HTTP</h4>
                     <code>Form</code> with operation type <code>invokeaction</code>
                 </td>
             </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_readall">
+                <td>
+                    <code>htv:methodName</code>
+                </td>
+                <td><code>GET</code></td>
+                <td>
+                    <code>Form</code> with operation type <code>readallproperties</code>
+                </td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_readmultiple">
+                <td>
+                    <code>htv:methodName</code>
+                </td>
+                <td><code>GET</code></td>
+                <td>
+                    <code>Form</code> with operation type <code>readmultipleproperties</code>
+                </td>
+            </tr>
+            <tr class="rfc2119-default-assertion" id="td-default-http-method_writeall">
```

Comment:
same here: in the second row I would add: "Form with operation type  writeproperty, writeallproperties, or writemultipleproperties"

Reply:
I agree, you should see the change in the new commit